src/commons/eventsourcing/store.rs

```diff
@@ -541,48 +563,6 @@ where
         Ok(CommandHistory::new(offset, total, commands))
     }
 
```

Comment:
Is archiving no longer done because archiving was a way to give operators a means to reduce the size of the on-disk store (they can choose whether to keep the older archived data or not) and now with this change the on-disk store should be less because less events are generated and stored?